pkg/connector/matrix.go

```diff
@@ -166,6 +167,23 @@ func (l *LinkedInClient) HandleMatrixMessage(ctx context.Context, msg *bridgev2.
 		}
 	}
 
+	if msg.ReplyTo != nil {
+		intent := l.userLogin.Bridge.Matrix.GhostIntent(l.userID)
+		evt, err := intent.GetEvent(ctx, msg.Event.RoomID, msg.ReplyTo.MXID)
+		if err != nil {
+			return nil, err
+		}
+		renderContent = append(renderContent, linkedingo.SendRenderContent{
+			RepliedMessageContent: &linkedingo.SendRepliedMessage{
+				OriginalSenderURN:  linkedingo.NewURN(string(msg.ReplyTo.SenderID)).WithPrefix("urn:li:msg_messagingParticipant:urn:li:fsd_profile"),
+				OriginalSendAt:     jsontime.UnixMilli{Time: msg.ReplyTo.Timestamp},
+				OriginalMessageURN: linkedingo.NewURN(string(msg.ReplyTo.ID)),
+				MessageBody: linkedingo.AttributedText{
+					Text: evt.Content.AsMessage().Body,
```

Comment:
This looks like it should be calling `matrixfmt.Parse` rather than just using the body?